plugin/skills/update-coding-guidelines/scripts/fetch-recent-review-comments.sh

```
@@ -0,0 +1,95 @@
+              isResolved isOutdated path line
+              comments(first: 50) {
+                nodes { author { login } body url createdAt }
+              }
+            }
+          }
+          comments(first: 100) {
+            nodes { author { login } body url createdAt isMinimized }
+          }
+        }
+      }
+    }' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)